Sitemap.php

```diff
@@ -164,6 +170,9 @@ private function createNewFile()
         if ($this->useXhtml) {
```

Comment:
It's better to extract writing whole XML header into separate protected method and override it in ImageSitemap. Sitemap base class should not know about images at all.